src/App.css

```diff
@@ -2,4 +2,231 @@
   padding: 0;
   margin: 0;
   box-sizing: border-box;
+  font-family: 'Montserrat', sans-serif;
+}
+
+body {
+  background: #f5f6fa;
+  font-size: 16px;
+}
+
+.panel-bg {
+  width: 100%;
+  height: 5.938rem;
+  padding: 1.438rem 6.188rem 1.688rem 6.25rem;
+  background-color: #fff;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  border-bottom: solid 1px #e8e8e8;
+}
+
+.panel-bg img {
+  height: auto;
+  width: 2.5rem;
+}
+
+.Bookstore-CMS {
+  margin-right: 2.938rem;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #0290ff;
+}
+
+nav {
+  display: inline-flex;
+  height: auto;
+}
+
+nav ul {
+  list-style: none;
+  display: flex;
```

Comment:
- [ ] The property below has been used several times in the CSS, to follow best practices of CSS - DRY(Don't Repeat Yourself) I suggest refactoring it to a class and applying the class in the HTML file to elements that share the same display property. Check this [article](https://speckyboy.com/good-bad-css-practices/) for more explanation.
```css
 display: flex;
```

Reply:
- [ ] Kindly check for other style repetitions and modify them.